### ISSUE TITLE
Inbox or Gmail?

### ISSUE THREAD

**Issue by okwme** [denysdovhan/inboxer#93]
I thought this issue clarified that support would be given to continue the inbox interface using the gmail API?
https://github.com/denysdovhan/inboxer/issues/77
I've just built the mac version of the newest release and it's gmail, not inbox. Do I have it backwards?

**Comment by jmarino** [comment]
You're right, the original plan laid out in issue #77 was to re-implement Inbox features on top of Gmail. As mentioned by @denysdovhan it will be a hard task and as far as I know there has been no progress on that front. Right now we are faced with a hard deadline at the end of March 2019 so we had to act.

I use Inboxer on a daily basis and would like to continue using it after the end of March even it means vanilla Gmail, that's why I adapted it to work with Gmail. We should continue working towards the goal of re-implementing Inboxer features, consider release 1.3.0 as a first step in that direction.

I'm gonna look into back porting some of the new features in 1.3.0 back to 1.2.1 and have another maintenance release that supports Inbox.
